src/components/calendar/CalendarView.jsx

```diff
@@ -33,29 +34,28 @@ const CalendarView = (props) => {
 
   // performs search with new "from" and "to" dates
   useEffect(() => {
-    axios.get(`${process.env.NEXT_PUBLIC_DOMAIN}/api/events?lng=${searchObj.lng}&lat=${searchObj.lat}&from=${currentStartDate}&to=${currentEndDate}&sort=time`)
+    axios.get(`${process.env.NEXT_PUBLIC_DOMAIN}/api/events?lng=${searchObj.lng}&lat=${searchObj.lat}&from=${fakeStartDate}&to=${fakeEndDate}&sort=time`)
       .then(res => {
-        // do we need a state for queryData?
-        setQueryData(res.data);
         return res.data;
       })
       .then(data => {
         const tempEventDates = new Set();
         data.features.forEach(event => {
-          const hasKeywords = searchObj.keywords.split(' ').every(keyword => {
-            if (event.properties.name.includes(keyword)
-              || event.properties.buskerName.includes(keyword)
-              || event.properties.description.includes(keyword)) {
+          const hasKeywords = searchObj.keywords.toLowerCase().split(' ').every(keyword => {
```

Comment:
It's odd that any of this is necessary, because that was supposed to be a job for the Search component, not this one.